syntax/blame.sublime-settings

```diff
@@ -0,0 +1,16 @@
+{
+	"always_show_minimap_viewport": false,
+	"auto_indent": false,
+	"detect_indentation": false,
+	"draw_indent_guides": false,
+	"draw_white_space": "None",
+	"font_options": [],
+	"gutter": false,
+	"line_numbers": false,
+	"margin": 20,
+	"match_brackets": false,
+	"match_tags": false,
+	"scroll_past_end": false,
```

Comment:
I would not merge this. I LOVE to scroll past end. That was to do we reading the information straight in front of my eyes rather than at the bottom of the screen.

Reply:
I always hated empty looking output panels caused by accidentally scrolling past end. They are located at the bottom of the screen anyway. 

Scolling past end of normal views might be a question of taste. As I didn't check which of the syntaxes is used in display output panels or normal views, I just copied the settings.

Reply:
That can be fixed by all ways focusing on line 1